### PR TITLE
0.17.1.1

### PR DESCRIPTION
**Pull request by mebagger**
October security update

### CODE REVIEW

File: src/chainparams.cpp [hunk 1]
```diff
@@ -127,8 +128,27 @@ class CMainParams : public CChainParams {
         nDefaultPort = 3333;
         nPruneAfterHeight = 100000;
 
+        // new parameters
+        consensus.DiffMode = 1;  //always 1
+        consensus.DiffModeV2 = 1334635; //set to hieght for the new diffmode
```
Comment (sumcoinlabs) [inline src/chainparams.cpp]: suggest POW = 1324010

File: src/chainparams.cpp [hunk 2]
```diff
@@ -127,8 +128,27 @@ class CMainParams : public CChainParams {
...
+        consensus.SubnHeight = 1334735;
+        consensus.SubV = 1385975;
+        consensus.SubBlks = 12;
+        consensus.nHeightEffective = 0;
+        consensus.nCoinbaseMaturity = 10;
+
+        // new tuneupconsensus parameters
+        tuneupconsensus = consensus;
+        tuneupconsensus.nHeightEffective = 1334535;  //fork height after this block for coinbase maturity
```
Comment (sumcoinlabs) [inline src/chainparams.cpp]: suggest coinbase maturity = 1323910

**Review by sumcoinlabs — CHANGES_REQUESTED** [review]

File: src/chainparams.cpp (hunk 1 above)
Comment (mebagger) [reply src/chainparams.cpp]: ok 
@mebagger
param update

File: src/chainparams.cpp (hunk 2 above)
Comment (mebagger) [reply src/chainparams.cpp]: ok 
@mebagger
param update

**Pull request merged**